core/src/trezor/messages.py

```
@@ -70,6 +70,7 @@ def __getattr__(name: str) -> Any:
     from trezor.enums import ThpMessageType  # noqa: F401
     from trezor.enums import ThpPairingMethod  # noqa: F401
     from trezor.enums import TronRawContractType  # noqa: F401
+    from trezor.enums import TronResourceCode  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)